modules/widgets/top-posts.php

```diff
@@ -488,12 +488,14 @@ function get_by_views( $count, $args ) {
 			$days = 2;
 		}
 
-		$post_view_posts = stats_get_csv( 'postviews', array( 'days' => absint( $days ), 'limit' => 11 ) );
-		if ( ! $post_view_posts ) {
+		$post_view_posts = stats_get_from_restapi( array(), 'top-posts?max=10&summarize=1&num=' . absint( $days ) );
```

Comment:
Thanks for the explanation! I've verified it works as you described so I set the number to 11 as before.